### ISSUE TITLE
Missing some commits?

### ISSUE THREAD

**Issue by davidyell** [codewithdary/hostinger-livewire-template#1]
Hey!

Thanks for the video tutorial, I have enjoyed watching through it. However I wanted to take a look at the completed project to see what the end result was.

It seems the `master` branch is missing some commits, and trying to setup the project causes errors as some of the files are missing.

![Screenshot from 2024-01-03 09-00-17](https://github.com/codewithdary/hostinger-livewire-template/assets/49889/462e8987-8120-420a-9525-155ef67e95b5)

Any chance you could push up the rest of the commits?

Thanks

**Comment by codewithdary** [comment]
Hi David!

Thank you for reaching out and expressing your enjoyment of the video tutorial. I'm glad to hear that you found it helpful!

I apologize for the inconvenience caused by the missing commits in the master branch and the errors encountered while setting up the project. It seems there was a mistake on my end, as I thought I had pushed the changes to the master branch. Unfortunately, I have deleted the complete project from my PC, making it impossible for me to update it further.

Best regards, 

Dary



On 3 January 2024 at 10:01:08, David Yell ***@***.***) wrote:

Thanks for the video tutorial, I have enjoyed watching through it. However I wanted to take a look at the completed project to see what the end result was. It seems the master branch is missing some commits, and trying to setup the project causes errors as some of the files are missing.
